packages/hint-stylesheet-limits/src/meta.ts

```diff
@@ -1,4 +1,4 @@
-import { Category } from 'hint/dist/src/lib/enums/category';
+import { Category } from '@hint/utils-types/dist/src/category';
```

Comment:
Sometimes we are using this form, others

```ts
import { Category } from '@hint/utils-types`;
```
Is there any particular reason when we use one or the other?

Reply:
If the package is used by the browser extension, sometimes to save a few bytes, some other times to avoid errors packing with webpack